src/network/networkd-dhcp-server.c

```diff
@@ -639,11 +639,19 @@ int config_parse_dhcp_server_emit(
                 if (r == 0)
                         return 0;
 
-                r = in_addr_from_string(AF_INET, w, &a);
-                if (r < 0) {
-                        log_syntax(unit, LOG_WARNING, filename, line, r,
-                                   "Failed to parse %s= address '%s', ignoring: %m", lvalue, w);
-                        continue;
+                if (streq(w, "_server_address"))
+                        a = IN_ADDR_NULL; /* null address will be converted to the server address. */
+                else {
+                        r = in_addr_from_string(AF_INET, w, &a);
+                        if (r < 0) {
+                                log_syntax(unit, LOG_WARNING, filename, line, r,
+                                           "Failed to parse %s= address '%s', ignoring: %m", lvalue, w);
+                                continue;
+                        }
+
+                        if (in4_addr_is_null(&a.in))
```

Comment:
Should this be documented in the manpage too?

Reply:
I noticed that `DNS=` in `[IPv6SendRA]` ignores a null address. I changed to this so.
And, I do not think it is not necessary to mention about the invalid address. There should be many invalid DNS server addresses, e.g. 255.255.255.255. Mentioning such addresses is overkill.